src/sidenav/components/SidenavGroupUI.vue

```diff
@@ -35,7 +35,10 @@
         </QBtn>
       </div>
     </template>
-    <SidenavMenu :entries="entries" />
+    <SidenavMenu
+      :entries="entries"
+      :entriesmore="entriesmore"
```

Comment:
Could this be `entriesMore` to keep to lowerCamelCase convention?

Reply:
Changing everything to `entriesMore` gives me a warning about the following rule: https://eslint.vuejs.org/rules/attribute-hyphenation.html

So I changed it to `:entries-more=entriesMore"`
Or should I just ignore the warning and have all lowerCamelCase without hyphenation?

Reply:
Yes, you got it! lowerCamelCase when in javascript, kebab-case when in a template :+1:

Reply:
Camels and kebabs... these things have funny names